tests/components/tolo/test_config_flow.py

```diff
@@ -23,6 +23,18 @@ def toloclient_fixture() -> Mock:
         yield toloclient
 
 
+@pytest.fixture
+def coordinator_toloclient() -> Mock:
+    """Patch ToloClient in async_setup_entry.
+
+    Throw exception to abort entry setup and prevent socket IO. Only testing config flow.
+    """
+    with patch(
+        "homeassistant.components.tolo.ToloClient", side_effect=Exception
+    ) as toloclient:
+        yield toloclient
```

Comment:
This doesn't change the test case. `ToloClient.__init__` already throws an exception when creating a new socket -> `SocketBlockedError`.

The `patch` just changes it so the `ToloClient` instance isn't created in the first place.